alert/index.js

```diff
@@ -0,0 +1,58 @@
+var stompClient = null;
+var id = null;
+
+$(document).ready(function(){
+    connect();
+
+    var urlParams = new URLSearchParams(window.location.search);
+    id = urlParams.get('id');
```

Comment:
Nothing. It will fail but we shouldn't show anything on the screen as this will be displayed on the user's stream. StreamLabs handles this the same way

Reply:
word. I was confused on the streaming behavior... We could do a better job with this later. But lets roll with this for now